services/s3/src/main/java/software/amazon/awssdk/services/s3/S3Configuration.java

```diff
@@ -272,6 +293,21 @@ public interface Builder extends CopyableBuilder<Builder, S3Configuration> {
 
         Boolean checksumValidationEnabled();
 
+        Boolean crossRegionBucketAccessEnabled();
+
+        /**
+         * Option to enable using cross region bucket Access.
+         * With this option client can access bucket of any region.
```

Comment:
Can we add more descriptions? 

v1 javadocs may be a good starting point https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-s3/src/main/java/com/amazonaws/services/s3/AmazonS3Builder.java#L352